package.json

```diff
@@ -100,6 +100,11 @@
       "!src/ui/**/*.js",
       "!src/index.js"
     ],
+    "moduleNameMapper": {
+      "^react-dnd$": "react-dnd/dist/cjs",
+      "^react-dnd-html5-backend$": "react-dnd-html5-backend/dist/cjs",
+      "^dnd-core$": "dnd-core/dist/cjs"
+    },
```

Comment:
why are these necessary?

Reply:
@ichalagashvili good question. this must be included when testing with react-dnd (this was already in appbuilder) - otherwise tests will fail. FYI, @entando/ddtable is now part of @entando/datatable (there's an option to make rows re-orderable on the new datatable so @entando/ddtable is needed). so we are planning to merge these both in the near future